app/assets/swagger/swagger.json

```diff
@@ -85,6 +85,117 @@
         }
       }
     },
+    "/courses/{course_id}/users": {
+      "parameters": [
+        {
+          "in": "path",
+          "name": "course_id",
+          "description": "id of course",
+          "required": true,
+          "schema": {
+            "type": "integer"
+          }
+        }
+      ],
+      "get": {
+        "summary": "List of users under the specificed course",
+        "tags": [
+          "courses", "users"
+        ],
+        "responses": {
+          "501": {
+            "description": "This endpoint has not yet been implemented yet",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/courses/{course_id}/add_user/{user_id}": {
+      "parameters": [
+        {
+          "in": "path",
+          "name": "course_id",
+          "description": "id of course",
+          "required": true,
+          "schema": {
+            "type": "integer"
+          }
+        },
+        {
+          "in": "path",
+          "name": "user_id",
+          "description": "id of user",
+          "required": true,
+          "schema": {
+            "type": "integer"
+          }
+        }
+      ],
+      "post": {
+        "summary": "Add an existing user to the specificed course",
+        "tags": [
+          "courses", "users"
+        ],
+        "responses": {
+          "501": {
+            "description": "This endpoint has not yet been implemented yet",
```

Comment:
i think the "not implemented" messages would be more useful if they confirmed which action was called, e.g. `The "add existing user to course" action is not implemented yet` or similar